zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1319,6 +1319,55 @@ paths:
             application/problem+json:
               schema:
                 $ref: "#/components/schemas/ProblemDetail"
+  /resources/{resourceKey}/deletion:
+    post:
+      tags:
+        - Resources
+        - Deletion
+      summary: Delete a resource
```

Comment:
```suggestion
      summary: Delete resource
```

Reply:
@mesellings if I add something like "Exclude articles from path `summary` names" to [the developer's guide](https://github.com/camunda/camunda/blob/main/docs/rest-controller.md#openapi-extension), does that capture this recommendation?

Reply:
@pepopowitz That sounds good, might be useful to add a few examples like this, and keep it as a guideline as there may be some longer endpoint names where we are better off including "a", "the" etc (can't think of any of the top of my head, but from the reviews I did there were some longer names than this one). Also, possibly tied into this is the convention of verb first and matching the verb in the description - e.g. Get resource, Gets the resource (rather than Obtains the resource for example . But perhaps that's for another day/comment!